test/ecto/changeset_test.exs

```diff
@@ -3696,7 +3665,13 @@ defmodule Ecto.ChangesetTest do
     end
 
     test "redacts all non-primary-key fields when schema sets @schema_redact :all_except_primary_keys" do
-      changeset = Ecto.Changeset.cast(%RedactAllExceptPrimaryKeysSchema{}, %{username: "Hunter", password: "hunter2"}, [:username, :password])
+      changeset =
+        Ecto.Changeset.cast(
+          %RedactAllExceptPrimaryKeysSchema{},
+          %{username: "Hunter", password: "hunter2"},
+          [:username, :password]
+        )
```

Comment:
this one seemingly was just caused by my `mix format`, sorry I can remove it if it's too annoying :)